### DISCUSSION TITLE
Can't setup Snapshot with emoji .eth domain?

### DISCUSSION THREAD

**Discussion by allanca** [snapshot-labs/snapshot#1509] [Q&A] [answered]
I'm trying to setup a new snapshot with an emoji-based eth domain. Everything seems to work well if there are no emojis in the name for me, but when I use an emoji domain, I can't get past setting the ENS text record.

Here's the domain I'm trying to setup: https://snapshot.org/#/%F0%9F%A5%93%F0%9F%AA%99%F0%9F%97%B3.eth/settings. It looks like there are errors in the JS console that may be related.

Any suggestions? For now, I'll stick with an ASCII domain.

**Comment by metrox-eth** [comment]


**Reply by ChaituVR** [reply]
Actually, Snapshot supports emojis and non-ASCII in ENS name, for example:
https://snapshot.org/#/%F0%9F%8D%AFdao.eth
https://snapshot.org/#/rad%C4%B1n.eth

**Comment by ChaituVR** [comment, marked as answer]
@allanca It should be now possible to setup snapshot space with emojis, please try and let me know if it doesn't work, 

Try going here https://snapshot.org/#/%F0%9F%A5%93%F0%9F%AA%99%F0%9F%97%B3.eth/settings
